src/braket/circuits/ascii_circuit_diagram.py

```diff
@@ -31,8 +31,13 @@
 class AsciiCircuitDiagram(CircuitDiagram):
     """Builds ASCII string circuit diagrams."""
 
-    @staticmethod
-    def build_diagram(circuit: cir.Circuit) -> str:
+    _vdelim = "|"
+    _qubit_line_char = "-"
+    _add_empty_line = True
+    _buffer_size = 0
+
+    @classmethod
+    def build_diagram(cls, circuit: cir.Circuit) -> str:
```

Comment:
Why does this need to be a `@classmethod`? Can't it still be a `@staticmethod`? Then it would still match the abstract method in `CircuitDiagram` base class, and no worries about backward incompatibility.

Probably changing `BoxDrawingCircuitDiagram` so that it inherits directly from `CircuitDiagram` would help here.